### PR TITLE
Add more SCA tooling

### PR DESCRIPTION
**Pull request by pxlpnk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ A curated list of awesome Node.js Security related resources.
 - [npq](https://github.com/lirantal/npq) - Safely install packages with npm or yarn by auditing them as part of your install process.
 - [snyk](https://www.npmjs.com/package/snyk) - Snyk helps you find, fix and monitor known vulnerabilities in Node.js npm, Ruby and Java dependencies, both on an ad hoc basis and as part of your CI (Build) system.
 - [node-release-lines](https://www.npmjs.com/package/node-release-lines) - Introspection API for Node.js release metadata. Provides information about release lines, their relative status along with details of each release.
+- [auditjs](https://github.com/OSSIndex/auditjs) - Audits an NPM package.json file to identify known vulnerabilities using the [OSSIndex](https://ossindex.sonatype.org/rest)
+- [npm-audit](https://docs.npmjs.com/cli/audit) - Runs a security audit based on your package.json
```
[the remark this answers is not in the document]
Comment (pxlpnk) [reply README.md]: I kind of assumed this was clear already based on the name :)

[the remark this answers is not in the document]
Comment (lirantal) [reply README.md]: Oh. Maybe it's just me then :-)
Looks great, let's merge!

[the remark this answers is not in the document]
Comment (pxlpnk) [reply README.md]: I updated it with some extra npm reference to be sure :)

[the remark this answers is not in the document]
Comment (lirantal) [reply README.md]: Cool :)

[the remark this answers is not in the document]
Comment (Elexy) [reply README.md]: LGTM.
Is [retirejs](https://github.com/RetireJS/retire.js) still a thing in this context?

[the remark this answers is not in the document]
Comment (pxlpnk) [reply README.md]: I think it could make sense. They state node.js on their GitHub project.

**Pull request merged**